PxWeb/Code/Api2/Serialization/XlsxDoubleColumnDataSerializer.cs

```diff
@@ -0,0 +1,17 @@
+﻿using Microsoft.AspNetCore.Http;
+using PCAxis.Paxiom;
+using PCAxis.Serializers;
+
+namespace PxWeb.Code.Api2.Serialization
+{
+    public class XlsxDoubleColumnDataSerializer : IDataSerializer
+    {
+        public void Serialize(PXModel model, HttpResponse response)
+        {
+            response.ContentType = "application/vnd.openxmlformats-officedocument.spreadsheetml.sheet; charset=" + System.Text.Encoding.Default.WebName;
```

Comment:
Content type is not needed for the binary formats for example Excel. You only need to set charset.

Reply:
It won't work if that part is left out. Will leave it as it is.

Reply:
It is only the encoding that can be omitted for the binary formats, you will still need the content-type.